testsuite/forge-cli/src/README.md

```diff
@@ -0,0 +1,36 @@
+---
+id: forge cli
+title: Forge CLI
+custom_edit_url: https://github.com/aptos-labs/aptos-core/edit/main/testsuite/forge-cli/README.md
+---
+
+# Forge CLI
```

Comment:
awesome! I'm planning on moving some of our internal docs to open source as well https://www.notion.so/aptoslabs/Forge-Runbook-80906ba523e2467e9efab9771caf9a91. It's mostly the k8s stuff, but we can combine into this README.

Reply:
Awesome! That would be great 😄